### PR TITLE
LUN-30: Generate mattermost notifications

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: .circleci/src/commands/notify.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+description: Send a notification (to Mattermost) at the end of a job, based on success or failure. Must be the last step in a job.
+parameters:
+  success_message:
+    default: ''
+    type: string
+  failure_mentions:
+    default: '@max.ammann @stefanie.metzger @sarah.sporck @steffen.kleinle'
```
Comment (f1sh1918) [inline .circleci/src/commands/notify.yml:7]: i think steffen added me recently to integreat app, i think it would make sense here, too

**Review by f1sh1918 — APPROVED** [review]
Looks good so far even i probably can not test it

File: .circleci/src/jobs/check.yml [hunk 2]
```diff
@@ -4,6 +4,7 @@ steps:
   - checkout
   - check_config
   - restore_yarn_cache
+  - restore_yarn_tools_cache
```
Comment (klinzo) [inline .circleci/src/jobs/check.yml:7]: Why do we need tools for checks?

File: .circleci/src/jobs/bump_version.yml [hunk 3]
```diff
@@ -12,15 +12,19 @@ steps:
   - restore_yarn_tools_cache
```
Comment (klinzo) [inline .circleci/src/jobs/bump_version.yml:12]: Perhaps it make sense to adjust this accordingly to integreat (change enum `delivery` to boolean `prepare_delivery` param

**Review by klinzo — APPROVED** [review]
Obviously not tested but it looks like you already did that. Only reviewed very superficial since most was probably copied from integreat.

File: .circleci/src/commands/notify.yml [hunk 4]
```diff
@@ -0,0 +1,79 @@
+description: Send a notification (to Mattermost) at the end of a job, based on success or failure. Must be the last step in a job.
```
[the remark this answers is not in the document]
Comment (SteffiStuffel) [reply .circleci/src/commands/notify.yml:1]: yes, mostly.

File: .circleci/src/jobs/check.yml (hunk 2 above)
Comment (SteffiStuffel) [reply .circleci/src/jobs/check.yml]: hmh, though without I got an error, will remove and test again, shouldn't be needed :thinking:

File: .circleci/src/jobs/bump_version.yml (hunk 3 above)
Comment (SteffiStuffel) [reply .circleci/src/jobs/bump_version.yml:12]: As this wasn't changed in integreat, when I was starting to work on this ticket, I would say out of scope, but if you think this change is important, feel free to open a ticket.

File: .circleci/src/jobs/check.yml (hunk 2 above)
Comment (SteffiStuffel) [reply .circleci/src/jobs/check.yml:7]: Oke, this is needed as ts:check also checks the tools-folder.

**Pull request merged**